src/mono/wasm/Wasm.Build.Tests/Wasm.Build.Tests.csproj

```diff
@@ -41,9 +41,12 @@
     <None Include="data\**\*" Link="data\%(RecursiveDir)%(FileName)%(Extension)" CopyToOutputDirectory="PreserveNewest" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Remove="..\testassets\WasmSatelliteLoading\words.ja-JP.resx" />
```

Comment:
Does it exist? We have asset `src\mono\wasm\testassets\resx\words.ja-JP.resx` but no `WasmSatelliteLoading` dir inside of `testassets`.